src/pymovements/utils/parsing.py

```diff
@@ -37,7 +37,7 @@
 @deprecated(
     reason='Please use gaze.from_asc() instead of utils.parsing. '
            'This module will be removed in v0.26.0.',
-    version='v0.21.0',
+    version='v0.21.1',
```

Comment:
why is `utils/parsing` deprecated in this PR?

Reply:
Parsing was already deprecated two weeks ago. #976 

What you see here is an autoreplace mistake. I had to change the version for all the current deprecation PRs because the PRs weren't merged for `v0.21.0`, so I had to replace the deprecations with `v0.21.1`.

I didn't notice that it also commited `parsing.py`. Good that you noticed!

I reverted this change.